.github/workflows/container-build.yml

```diff
@@ -0,0 +1,33 @@
+# Build containers on every PR
+# See also container-release.yml
+
+name: Container Build
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  build-container:
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        GO_VERSION:
+          - "1.23.4"
```

Comment:
Please add a reference to these files to [this comment in tag_and_upload](https://github.com/letsencrypt/boulder/blob/2678e688060e91af545cb6055cdb22c156e087cb/test/boulder-tools/tag_and_upload.sh#L10-L14).